src/store/editor/reducer.js

```diff
@@ -89,6 +93,7 @@ const DEFAULT_STATE = {
 		patterns: [],
 		defaultPreferences: {},
 		allowApi: false,
+		disableCanvasAnimations: false,
```

Comment:
For our Popup preview use case, we need to use a `position: absolute` on the preview.
We faced a weird behavior that seemed to be related to the animation library.
Some of the styling properties were kept when switching between previews, causing an issue when switching from an absolute positioned preview to one of the other.
Disabling the animations solved this issue.